src/Command/SearchImportCommand.php

```diff
@@ -62,8 +87,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $entitiesToIndex = array_unique($entitiesToIndex);
 
         foreach ($entitiesToIndex as $entityClassName) {
-            $manager    = $this->managerRegistry->getManagerForClass($entityClassName);
-            $repository = $manager->getRepository($entityClassName);
+            $manager       = $this->managerRegistry->getManagerForClass($entityClassName);
+            $repository    = $manager->getRepository($entityClassName);
+            $realIndexName = $this->searchService->searchableAs($entityClassName);
+
+            if ($shouldDoAtomicReidex) {
+                $tmpIndex = $this->searchServiceForAtomicReindex->searchableAs($entityClassName);
```

Comment:
you can just change the $tmpIndex here and add `tmp_` or any name. That way you don't have to register the `SearchService` two times

Reply:
I do, because of `$indexingService->index()` call